### ISSUE TITLE
Error when closing the main RoboCJK window

### ISSUE THREAD

**Issue by justvanrossum** [BlackFoundryCom/robo-cjk#34]
Steps to reproduce:
- Fresh start of RF
- Start RoboCJK with start.py
- Login, choose project, choose font
- Open a CG for editing
- Close CG
- Close main RoboCJK window

Closing the main RoboCJK window prints a double traceback, as seen below:

Whole session output:
```
hello
'glyphs_composition_database'
total to load 0.36855602264404297
download glyphs: 0.4906461238861084 seconds to download uni34D3
insert glyphs: 0.25344276428222656 seconds to insert 11
get locked-by: 0.0830538272857666 seconds to get locked-by for uni34D3
calculate preview: 0.38976573944091797 seconds to calculate preview of uni34D3
display glyph: 1.2327070236206055 seconds to display uni34D3
download glyphs: 0.5203819274902344 seconds to download uni34D3
insert glyphs: 0.28034400939941406 seconds to insert 11
download glyphs: 0.4242069721221924 seconds to download uni34D3
insert glyphs: 0.282210111618042 seconds to insert 11
download glyphs: 0.5314168930053711 seconds to download uni34D3
insert glyphs: 0.383991003036499 seconds to insert 11
3.346619129180908 to open a uni34D3
Unlocked glyphs: uni34D3
1.0351390838623047 to close uni34D3
Traceback (most recent call last):
  File "/Applications/RoboFont-3.5b.app/Contents/Resources/lib/python3.7/vanilla/vanillaWindows.py", line 604, in windowWillClose_
    self._alertBindings("close")
  File "/Applications/RoboFont-3.5b.app/Contents/Resources/lib/python3.7/vanilla/vanillaWindows.py", line 596, in _alertBindings
    value = callback(self)
  File "/Users/just/code/git/BlackFoundry/robo-cjk/sources/views/roboCJKView.py", line 935, in windowCloses
    self.currentFont.saveGlyph(self.currentGlyph)
  File "/Users/just/code/git/BlackFoundry/robo-cjk/sources/models/font.py", line 1075, in saveGlyph
    self.client.character_glyph_update(self.uid, glyph.name, xml)
  File "/Users/just/code/git/BlackFoundry/robo-cjk/sources/controllers/client.py", line 815, in character_glyph_update
    return self._api_call('character_glyph_update', params)
  File "/Users/just/code/git/BlackFoundry/robo-cjk/sources/controllers/client.py", line 113, in _api_call
    raise HTTPError(f"{response.status_code} {response_data['error']}")
controllers.client.HTTPError: 403 Forbidden - Character Glyph object must be locked by the current user.
Traceback (most recent call last):
  File "lib/doodleDelegate.pyc", line 96, in sendEvent_
  File "/Applications/RoboFont-3.5b.app/Contents/Resources/lib/python3.7/vanilla/vanillaWindows.py", line 604, in windowWillClose_
    self._alertBindings("close")
  File "/Applications/RoboFont-3.5b.app/Contents/Resources/lib/python3.7/vanilla/vanillaWindows.py", line 596, in _alertBindings
    value = callback(self)
  File "/Users/just/code/git/BlackFoundry/robo-cjk/sources/views/roboCJKView.py", line 935, in windowCloses
    self.currentFont.saveGlyph(self.currentGlyph)
  File "/Users/just/code/git/BlackFoundry/robo-cjk/sources/models/font.py", line 1075, in saveGlyph
    self.client.character_glyph_update(self.uid, glyph.name, xml)
  File "/Users/just/code/git/BlackFoundry/robo-cjk/sources/controllers/client.py", line 815, in character_glyph_update
    return self._api_call('character_glyph_update', params)
  File "/Users/just/code/git/BlackFoundry/robo-cjk/sources/controllers/client.py", line 113, in _api_call
    raise HTTPError(f"{response.status_code} {response_data['error']}")
controllers.client.HTTPError: 403 Forbidden - Character Glyph object must be locked by the current user.
```

**Comment by GaetanBaehr** [comment]
this is fixed db320f5e0e88084cba2b1eb47f0828a9c2ca6c5e

**Comment by justvanrossum** [comment]
Thanks!
